player/command.c

```diff
@@ -3801,7 +3813,7 @@ static const char *const *const mp_event_property_change[] = {
       "demuxer-cache-time", "cache-buffering-state", "cache-speed",
       "demuxer-cache-state"),
     E(MP_EVENT_WIN_RESIZE, "current-window-scale", "osd-width", "osd-height",
-      "osd-par", "osd-dimensions"),
+      "osd-par", "osd-dimensions", "window-id"),
```

Comment:
AFAIK, the window id will only be set on window creation and won't change afterwards. But I don't see any "winopen"-ish event so just doing it every resize.

Reply:
You can just omit adding it here since the property is not expected to change.

Reply:
Hmm, weird. Without this "window-id" used to be always nil, but it's working fine now. So I probably messed something up during previous testing (most likely I was accidentally testing with a stale build).